.changeset/short-walls-hear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
```

Comment:
This is technically major, but the breaking change affects a feature we're not using anywhere yet?